src/midiEditorCommands.cpp

```diff
@@ -500,4 +500,23 @@ void cmdFocusNearestMidiEvent(Command* command) {
 		}
 	}
 }
+
+static BOOL CALLBACK cbFindFilterWindow(HWND hWnd, LPARAM lParam) {
+	WCHAR szText[128];
+	if ((GetWindowTextW(hWnd, szText, 128) != 0) &&
+		!wcsncmp(szText, L"Filter Events", 13)) {
+		*((HWND *)lParam) = hWnd;
+		return false;
+	}
+	return true;
+}
+
+void cmdMidiFilterWindow(Command *command) {
+	HWND editor = MIDIEditor_GetActive(), filter = NULL;
```

Comment:
nit:

1. Please split this onto two separate lines, one variable per line.
2. Please use nullptr instead of NULL.

Reply:
I have not seen any nullptr in this file ;)
BTW: when compiling using default VS2015 settings, several  "deprecated" options are required, to avoid failures from indirect conversions.

Reply:
> I have not seen any nullptr in this file ;)

That's correct. :) I've become a lot more familiar with modern C++ since I started working on OSARA. I'm ensuring newer code uses modern conventions, but some of the older code doesn't.

> BTW: when compiling using default VS2015 settings, several "deprecated" options are required, to avoid failures from indirect conversions.

Happy to take a PR if those changes are within OSARA. If the issues are in REAPER or WDL headers, though, that's out of scope for OSARA.